addons/survey/controllers/main.py

```diff
@@ -549,3 +552,30 @@ def _extract_filters_data(self, survey, post):
         user_input_lines = request.env['survey.user_input'].sudo().search(user_input_domain).mapped('user_input_line_ids')
 
         return user_input_lines, search_filters
+
+    @http.route('/survey/background/<int:survey_id>', type='http', auth="public", website=True, sitemap=False)
+    def survey_get_background(self, survey_id):
+        survey = request.env['survey.survey'].sudo().browse(survey_id).exists()
```

Comment:
We could improve this method to avoid code copy paste and avoid desynchronization.

For example

  * add answer token parameter
  * check like other routes if it is valid
  * call ir_http.sudo().binary_content() is good as standard code handles it
  * in web/main.py we could split _content_image, move the part after call to binary_content in _content_image_get_response (separate commit)
  * call _content_image_get_response() and boom !